### ISSUE TITLE
Is it possible to increase the menu font size in the Windows version of Geany 2.0 WITHOUT recompiling the program?

### ISSUE THREAD

**Issue by dlrourke** [geany/geany#3652]
This is more a question than a bug report.

First, thank you very, very much for the new (2.0) version of Geany!
As my eyes aren't what they used to be, is it possible to increase the menu font size in the Windows version of Geany WITHOUT recompiling it.  An additional two points of font size would be great.

Thanks.

**Comment by b4n** [comment]
Yes, you can adjust this with custom [GTK CSS](https://docs.gtk.org/gtk3/css-overview.html).  Go to [*Tools → Configuration Files → geany.css*](https://geany.org/manual/#customizing-geany-s-appearance-using-gtk-css), and insert something like `menu { font-size: 15pt; }` or `* { -gtk-dpi: 120; }`, save and restart Geany.

You might be interested in [GTK CSS properties](https://docs.gtk.org/gtk3/css-properties.html) as well.
